src/core/modules/impl/ArweaveGatewayInteractionsLoader.ts

```diff
@@ -142,5 +145,12 @@ export class ArweaveGatewayInteractionsLoader implements InteractionsLoader {
 
   set warp(warp: Warp) {
     this.arweaveTransactionQuery = new ArweaveGQLTxsFetcher(warp);
+    this._warp = warp;
   }
 }
+
+export function hasVrfTag(interaction: GQLNodeInterface) {
+  return interaction.tags.some((t) => {
+    return t.name == WARP_TAGS.REQUEST_VRF && t.value === 'true';
+  });
+}
```

Comment:
hmm, rest of the functions like this one (`isInteractWrite`, `isEvmSigned`) are in the TagsParser class. Maybe let's stick to the convention. Seems more like a generic function but I leave it to you to decide.

Reply:
done!